<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeGeneratorOpenCLwithGUIClass</class>
 <widget class="QMainWindow" name="CodeGeneratorOpenCLwithGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>946</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CodeGeneratorOpenCLwithGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>591</x>
      <y>752</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>532</y>
      <width>361</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>-9</x>
      <y>502</y>
      <width>851</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="FIFO_SIZE">
    <property name="geometry">
     <rect>
      <x>523</x>
      <y>626</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputDirInput">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>222</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="convertButton">
    <property name="geometry">
     <rect>
      <x>451</x>
      <y>692</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nativeFileSelect_2">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>362</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outputDirSelect">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>222</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="networkFileInput">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>32</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="nativeFileSelect_1">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>322</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-9</x>
      <y>182</y>
      <width>821</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="networkFileSelect">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>32</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>-9</x>
      <y>272</y>
      <width>811</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-9</x>
      <y>82</y>
      <width>811</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="Target_OpenCL">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>570</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Use OpenCL</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>454</x>
      <y>625</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>FIFO Size:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nativeFileInput_4">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>442</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="No_OpenCL">
    <property name="geometry">
     <rect>
      <x>451</x>
      <y>542</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Without OpenCL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>92</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Source Directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nativeFileSelect_3">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>402</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>272</y>
      <width>601</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Native C Files(optional) - Header will be created for all given files, but imports cannot be checked!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nativeFileInput_3">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>402</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="nativeFileSelect_4">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>442</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nativeFileInput_2">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>362</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nativeFileInput_1">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>322</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>12</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Network File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>182</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sourceDirSelect">
    <property name="geometry">
     <rect>
      <x>641</x>
      <y>132</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SourceDirInput">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>132</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CodeGeneratorOpenCLwithGUI.qrc"/>
 </resources>
 <connections/>
</ui>
